core/PRP/Geometry/plGBufferGroup.h

```diff
@@ -90,6 +93,12 @@ class PLASMA_DLL plGBufferGroup
         kStoreIsDirty = 0x4
     };
 
+    enum
+    {
+        kMaxNumVertsPerBuffer = 32000,
+        kMaxNumIndicesPerBuffer = 32000,
```

Comment:
I feel like "Num" is redundant here...

Reply:
Probably so, but I copy-pasta'd this directly from Plasma. While we at it, maybe the values should be changed to INT16_MAX?